### PR TITLE
instructions for how to build local documentation

### PR DESCRIPTION
**Pull request by lsmor**
Plutus API documentation is actually fairly OK.

### CODE REVIEW

**Review by nstankov-bg — CHANGES_REQUESTED** [review]
Hey @lsmor,

Can we maybe separate that in its own item.
If it's in NIX, it should be available outside of Ubuntu.

**Comment by lsmor** [comment]
Hi @nstankov-bg , 

Moved documentation build as a new item in the setup + added some additional information about using the docs.

File: book/SUMMARY.md [hunk 1]
```diff
@@ -8,6 +8,16 @@
   * [Windows WSL2](/docs/setup/WindowsWSL.md)
   * [Ubuntu](/docs/setup/Ubuntu.md)
   * [MacOS including M1](/docs/setup/MacOS.md)
-
+  * [Local Plutus API documentation (Optional)](/docs/setup/buildDocumentation.md)
```
Comment (nstankov-bg) [inline book/SUMMARY.md]: CC: @Dauta  and @dino- .

Would that not be a better fit under "Additional Configuration", instead of Getting Started?

Comment (Dauta) [reply book/SUMMARY.md]: Yeah, I'd say so as well. Additional or miscellaneous.

Comment (lsmor) [reply book/SUMMARY.md]: Sorry, first time contributing on Github. Moved under Additional Configuration

**Review by nstankov-bg — APPROVED** [review]
Awesome effort, thank you for exploring that!

**Pull request merged**